sklearn/conftest.py

```diff
@@ -37,6 +38,17 @@
     "fetch_rcv1_fxt": fetch_rcv1,
 }
 
+_SKIP32_MARK = pytest.mark.skipif(
+    environ.get("SKLEARN_SKIP_FLOAT32", "1") != "0",
+    reason="Set SKLEARN_SKIP_FLOAT32=0 to run float32 dtype tests",
+)
```

Comment:
How about using this phrasing and semantic?
```suggestion
_SKIP32_MARK = pytest.mark.skipif(
    environ.get("SKLEARN_RUN_FLOAT32", "0") != "1",
    reason="Set SKLEARN_RUN_FLOAT32=1 to run float32 dtype tests",
)
```

I find it more intuitive. What do you think?

Reply:
I like you suggestion. In general, I think the "positive" version with "SKLEARN_RUN_*` is easier to reason about.